kubernetes/Earthfile

```
@@ -1,15 +1,28 @@
 start:
     LOCALLY
     
-    # Ensure local dependencies are present
-    RUN which kind && which kubectl
+    DO +DEPS
 
     RUN kind create cluster --name earthlydemo
 
-    RUN kubectl apply -f buildkit.yaml
-    RUN kubectl wait --for=condition=ready pod -l component=buildkitd -l app=earthly --timeout=90s
+    RUN kubectl apply -f ./buildkit.yaml
+    RUN kubectl wait --for=condition=available deployment/earthly-buildkitd              --timeout 90s
```

perhaps it's optional, but this would make it more consistent with the other commands
```suggestion
    RUN kubectl wait --for=condition=available deployment/earthly-buildkitd              --timeout=90s
```